docs/source/en/using-diffusers/callback.md

```diff
@@ -31,9 +31,9 @@ def callback_dynamic_cfg(pipe, step_index, timestep, callback_kwargs):
                 prompt_embeds = callback_kwargs["prompt_embeds"]
                 prompt_embeds = prompt_embeds.chunk(2)[-1]
 
-        # update guidance_scale and prompt_embeds
-        pipe._guidance_scale = 0.0
-        callback_kwargs["prompt_embeds"] = prompt_embeds
+                # update guidance_scale and prompt_embeds
+                pipe._guidance_scale = 0.0
+                callback_kwargs["prompt_embeds"] = prompt_embeds
```

Comment:
What's this change about?

Reply:
I ran this sample code and got `local variable 'prompt_embeds' referenced before assignment` so I think it would make sense if we move those lines inside `if` block